templates/customernotification.html

```diff
@@ -10,73 +10,46 @@
-                            
-                            {{ $FirstAlert = false }}
-                        {{ end }}
-                        {{ if and (eq .Value "PersistentVolumeUsageNearFull" ) (eq $FirstAlert true ) }}
-                            Persistent Volume Usage is Nearly Full
-                            <br><br>
-                            The utilization of one or more of the PVs in your cluster has exceeded 75%. Please free up some space or expand the PV if possible. Failure to address this issue may lead to service interruptions.
-                            
-                            {{ $FirstAlert = false }}
-                        {{ end }}
-                        {{ if and (eq .Value "PersistentVolumeUsageCritical" ) (eq $FirstAlert true ) }}
-                            Persistent Volume Usage Critical
-                            <br><br>
-                            The utilization of one or more of the PVs in your cluster has exceeded 85%. Please free up some space immediately or expand the PV if possible. Failure to address this issue may lead to service interruptions.
-                            
-                            {{ $FirstAlert = false }}
-                        {{ end }}
-                        {{ if and (eq .Value "CephClusterReadOnly" ) (eq $FirstAlert true ) }}
-                            Ceph Cluster is in Read Only state
-                            <br><br>
-                            Your storage cluster utilization has crossed 85%, as a result, access is now in a read-only state! Please free up some space or if possible expand the storage cluster immediately to prevent any additional service access issues.
-                            
-                            {{ $FirstAlert = false }}
-                        {{ end }}
-                        {{ if and (eq .Value "CephClusterNearFull" ) (eq $FirstAlert true ) }}
-                            Ceph Cluster is Nearly Full
-                            <br><br>
-                            Your storage cluster utilization has crossed 75% and will move into a read-only state at 85%. Please free up some space or if possible expand the storage cluster immediately to prevent any service access issues.
-
-                            {{ $FirstAlert = false }}
-                        {{ end }}
-                        <br>
-                {{ end }}
-                <ul>
-                    {{if eq .Name "persistentvolumeclaim" }}
-                        <li>PVC Name: {{ .Value }}</li>
-                    {{ end }}
-                    {{if eq .Name "namespace" }}
-                        <li>Namespace: {{ .Value }}</li>
-                    {{ end }}
-                </ul>
-            {{ end }}
+            <ul>    
+                {{ with .Labels.persistentvolumeclaim }}<li>PVC Name: {{ . }}</li>{{ end }}
+                {{ with .Labels.persistentvolumeclaim }}<li>Namespace: {{ .Labels.namespace }}</li>{{ end }}
```

Comment:
I this context `.` will be `.Labels.persistentvolumeclaim` so `.Labels.namespace` will become `.Labels.persistentvolumeclaim.Labels.namespace` which I believe is not what you intendant 

The solution will be to use an `if` the encompass the entire UL based on the value of `.Labels.persistentvolumeclaim`